demo/src/cartesian.cpp

```diff
@@ -108,16 +108,8 @@ Task createTask() {
 		t.add(std::move(stage));
 	}
 
-	{  // move gripper into predefined open state
```

Comment:
I removed these lines since we don't have a controller for the gripper, yet

Reply:
could you comment this and add a TODO instead?